app/styles/main.css

```diff
@@ -0,0 +1,84 @@
+body {
+    font-family: 'Roboto', sans-serif;
+    /*
+    Thin 100
+    Regular 400
+    */
+  }
+  body:before{
+    background-image: url("/assets/images/tswirls.png");
+    content: '';
+    display: block;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: -1;
+  }
+  
+  
+  .welcome-title {
+    margin-top: 70px;
+    text-align: center;
+    font-weight: 400;
+  }
+  
+  .logo-img {
```

Comment:
I would suggest looking into [BEM css methodology](http://getbem.com/naming/) for making sure that a change in the sequence of your imports in the app.css do not cause a UI break in the widths of your images/text